data/is_ocaml_yet/gui.md

```diff
@@ -0,0 +1,74 @@
+---
+id: gui
+question: Is OCaml GUI Yet?
+answer: Not quite yet, but we're getting there!
+categories:
+  - name: Frameworks
+    status: 🟡
+    description: |
+      These are libraries written entirely in OCaml. It has packages for syndicating content and working with RSS feeds.
```

Comment:
```suggestion
      These are libraries written entirely in OCaml. It has packages for syndicating content and working with RSS feeds.
```

RSS feed syndication doesn't make sense here, and yes, the libraries are written entirely in OCaml but why would I care? I mean what's this section about?